pkg/netpol/eval/check.go

```diff
@@ -170,12 +176,12 @@ func (pe *PolicyEngine) allallowedXgressConnections(src, dst k8s.Peer, isIngress
 	// relevant policies: policies that capture dst if isIngress, else policies that capture src
 	var netpols []*k8s.NetworkPolicy
 	if isIngress {
-		if dst.PeerType == k8s.Iptype {
+		if dst.PeerType == k8s.IPBlockType {
```

Comment:
may want to consider making `peer` an interface/struct since there seem to be variations on it